CHANGELOG.md

```diff
@@ -7,6 +7,9 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## 39.2.3
```

Comment:
New additions to the changelog need to be added under the heading `Unreleased`, see https://github.com/alphagov/govuk_publishing_components/blob/main/docs/publishing-to-rubygems.md

Reply:
Thanks @AgaDufrat

Reply:
Updated as requested but I’m keen to understand the reasons for that. It seems rather unproductive to have to create a new PR just to publish the gem. 

Platform Security and Reliability is trying to improve our gem release process by introducing automation and encouraging engineers to release the changes in the same PR. This would be very valuable insight for this piece of work.

Reply:
Yeah, I can see that. Sorry. The reasoning behind our release process hasn't been documented, but I'd imagine:

- it helps us to minimise the number of releases that happen, by bundling changes together (we're also encouraged to ask others if they have PRs to merge first when doing a release)
- it prevents two PRs from containing the same version number when merging (you'd hope a merge conflict would happen to prevent this, but sometimes it doesn't)
- it gives a clearer history of the releases

It is something that adds a bit of friction to our workflow, so if you have any corresponding insight, I'm sure the frontend team would be up for a discussion.

Reply:
Thank you! Very useful to know.